browser-test/src/civiform_admin_program_statuses.test.ts

```diff
@@ -0,0 +1,189 @@
+import {
+  dismissModal,
+  startSession,
+  logout,
+  loginAsAdmin,
+  AdminPrograms,
+  AdminProgramStatuses,
+} from './support'
+import {Page} from 'playwright'
+
+describe('modify program statuses', () => {
+  let pageObject: Page
+  let adminPrograms: AdminPrograms
+  let adminProgramStatuses: AdminProgramStatuses
+
+  beforeAll(async () => {
+    const {page} = await startSession()
+    pageObject = page
+    adminPrograms = new AdminPrograms(pageObject)
+    adminProgramStatuses = new AdminProgramStatuses(pageObject)
+
+    await loginAsAdmin(pageObject)
+  })
+
+  describe('initial status list', () => {
```

Comment:
nit: the phrasing is confusing me a little....

maybe: "New  program creation"

Reply:
Done!